JavaScript/JavaScriptSDK/TelemetryContext.ts

```diff
@@ -92,14 +93,15 @@ module Microsoft.ApplicationInsights {
                 this.session = new Context.Session();
                 this.sample = new Context.Sample(config.sampleRate());
             }
+
+            this._addDefaultTelemetryInitializers();
```

Comment:
You can, but it will hacky ;/ telmetryInitializers array is private and is not exposed in d.ts, functions are usually hard to compare and it will be difficult to find it among other initializers.